Source/Internal/API/APIRequest.swift

```diff
@@ -26,11 +26,19 @@ import Foundation
 internal typealias JSONObject = [String: Any]
 internal typealias JSONArray = [JSONObject]
 
+/// This enum contains supports HTTP request methods
```

Comment:
@akueisara I think this sentence is not grammatically correct `contains supports`

Reply:
Ah yes, it should be 'supported'. Thanks.